api/app/Console/Commands/GenerateKeysCommand.php

```diff
@@ -0,0 +1,62 @@
+<?php namespace App\Console\Commands;
+
+use Crypt_RSA;
+use Illuminate\Console\Command;
+use Illuminate\Filesystem\Filesystem;
+
+class GenerateKeysCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'spira:generate-keys';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Generate RSA key pair for the application.';
+
+    /**
+     * Filesystem.
+     *
+     * @var Illuminate\Filesystem\Filesystem
+     */
+    protected $file;
+
+    /**
+     * Create a new command instance.
+     *
+     * @param  Illuminate\Filesystem\Filesystem  $file
+     * @return void
+     */
+    public function __construct(Filesystem $file)
+    {
+        parent::__construct();
+
+        $this->file = $file;
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $rsa = new Crypt_RSA();
+        extract($rsa->createKey());
```

Comment:
Can you refactor this to basic assignment for better readbility? I read the below `$privatekey` var and couldn't immediately work out where it came from.